### ISSUE TITLE
OKHsl is on docs but actually not in the published version

### ISSUE THREAD

**Issue by leadream** [color-js/color.js#514]
I saw OKHsl space is on the doc but I can't use it in the latest version, which confused me. So when do you plan publish it to npm? Or could you mark it as unpublished in the docs? Thanks
https://colorjs.io/docs/spaces#okhsl

<img width="629" alt="image" src="https://github.com/color-js/color.js/assets/12868055/a5b430c1-a96f-4bdc-b8af-204751a8b32c">

**Comment by facelessuser** [comment]
I agree that I think it would be appropriate to tag a new release, but I am not usually involved in that process. I think there is a significant enough number of changes and fixes to warrant a new release.

**Comment by LeaVerou** [comment]
> I agree that I think it would be appropriate to tag a new release, but I am not usually involved in that process. I think there is a significant enough number of changes and fixes to warrant a new release.

Yes absolutely. And [we promised a new release each month in v0.5.0](https://github.com/color-js/color.js/releases/tag/v0.5.0). That was 2.5 months ago 😅

**Comment by TheAleSch** [comment]
still waiting for it in June.. almost July 😓

**Comment by svgeesus** [comment]
We are, slowly, working on a new release. We know it is late; some real-life issues slowed us down.

 - https://github.com/color-js/color.js/issues/537

**Comment by mevbg** [comment]
> We are, slowly, working on a new release. We know it is late; some real-life issues slowed us down.
> 
> 
> 
>  - https://github.com/color-js/color.js/issues/537

Any progress with this one, @svgeesus?
I'd really want to work with OkHSL. 🙂

**Comment by facelessuser** [comment]
Isn't the latest version [v0.6.0-alpha.1](https://github.com/LeaVerou/color.js/releases/)? You should be able to grab Okhsl in that release.
